inc/commonitilvalidation.class.php

```diff
@@ -861,7 +876,22 @@ function showSummary(CommonDBTM $item) {
             echo "<td>".$row["comment_submission"]."</td>";
             echo "<td>".Html::convDateTime($row["validation_date"])."</td>";
             echo "<td>".getUserName($row["users_id_validate"])."</td>";
-            echo "<td>".$row["comment_validation"]."</td>";
+            echo "<td>".$this->getValueToDisplay(2, $row["comment_validation"],
+                                                   ['html' => true])."</td>";
+
+            $doc_item = new Document_Item();
+            $docs = $doc_item->find(["itemtype" => $this->getType(), "items_id" => $this->getID()]);
```

Comment:
Mabe you should add `"timeline_position" => ['>', CommonITILObject::NO_TIMELINE]` in criteria. Without this, pasted images in rich text appears in the document list.

Reply:
done